### ISSUE TITLE
Add ServiceInsight and ServicePulse

### ISSUE THREAD

**Issue by udidahan** [Particular/Workshop.Microservices#89]
In the last delivery of the workshop at NDC Oslo 2017, I instructed attendees to look at ServiceInsight as a part of the exercises that they were doing, as well as adding the Saga Audit plugin for the saga exercises.

For the existing NServiceBus users/customers who attended, experiencing Pulse and Insight first-hand was very significant. I heard them saying that there was no way that they'd go back to working without them, almost regardless of how much they cost.

In short, I propose that we incorporate more instruction into the workshop around how to install, configure, and use Pulse and Insight in the workshop.

I'd also suggest that this be prioritized so that it can be done in time for the next delivery of the workshop which, I believe, is Jimmy doing it in NDC Sydney.
